css/app.css

```diff
@@ -479,7 +480,7 @@ header.hide-controls #window-close {
   min-height: calc(100% - 2*var(--editor-gutter-vertical-padding));
   height: auto;
   padding: var(--editor-gutter-padding);
-  min-width: 29px;
+  min-width: calc(29px - var(--editor-gutter-horiztonal-padding));
```

Comment:
pull the 29 out into a var too for consistency, and where the vars are defined add a comment about how they are directly copied from codemirror textarea to match formatting with our custom textarea

Reply:
Done